lib/capability/capability.go

```diff
@@ -94,8 +107,13 @@ func setKnownCapabilities(config *configv1.ClusterVersion) map[configv1.ClusterV
 }
 
 // setEnabledCapabilities populates a map keyed by capability from all enabled capabilities as defined in ClusterVersion.
-// DefaultCapabilitySet is used if a baseline capability set is not defined by ClusterVersion.
-func setEnabledCapabilities(config *configv1.ClusterVersion) map[configv1.ClusterVersionCapability]struct{} {
+// DefaultCapabilitySet is used if a baseline capability set is not defined by ClusterVersion. A check is then made to
+// ensure that no previousily enabled capability is now disabled and if any such capabilities are found each is enabled,
+// saved, and returned.
+func setEnabledCapabilities(config *configv1.ClusterVersion,
+	priorEnabled map[configv1.ClusterVersionCapability]struct{}) (map[configv1.ClusterVersionCapability]struct{},
```

Comment:
It feels a bit strange to pass through both `priorEnabled` and a `*configv1.ClusterVersion` (whose `status.capabilities` will also include a set of prior, enabled caps.  To make the distinction clear, can we adjust the function signature here to take `*configv1.ClusterVersionCapabilitiesSpec` and `priorEnabled`?